app-starter/static/app-conf-projected.json

```diff
@@ -86,9 +86,8 @@
       "projection": "EPSG:28992",
       "tileGridRef": "dutch_rd",
       "attributions": "<a href='https://www.pdok.nl' target='_blank'>PDOK</a> by Dutch Kadaster",
-      "isBaseLayer": false,
-      "visible": false,
-      "displayInLayerList": true
+      "isBaseLayer": true,
```

Comment:
This seems not be an appropriate base layer, this has more an overlay character, so I vote set `isBaselayer: false` here